src/helpers/tokens/generate.token.js

```diff
@@ -2,11 +2,9 @@ import jwt from 'jsonwebtoken';
 
 const createToken = (user, secret) => {
   try {
-    const token = jwt.sign({ user },
-      secret,
-      {
-        expiresIn: '24h'
-      });
+    const token = jwt.sign({ user }, secret, {
```

Comment:
@nkpremices why does everyone keep modifying the syntax in this file?

Reply:
This one is because of the ```rebase``` we are all doing. There is a mistake I think that we were doing. We've been merging our local branches. That affected our workflow. But now we've decided to do not do that again. This kind of errors won't happen again @MemunaHaruna